browser/rewards_data/counters/rewards_counter.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_REWARDS_DATA_COUNTERS_REWARDS_COUNTER_H_
+#define BRAVE_BROWSER_REWARDS_DATA_COUNTERS_REWARDS_COUNTER_H_
+
+#include "base/macros.h"
+#include "base/memory/weak_ptr.h"
+#include "brave/components/rewards_data/core/counters/rewards_data_counter.h"
+
+class Profile;
+
+namespace rewards_counter {
+  using OnRewardsCountedCallback =
+      base::Callback<void(int64_t, base::string16)>;
+}
+
+// A RewardsDataCounter that counts the number of Rewards
+// Auto-Contribute sites as seen on the brave://settings#clearRewardsData page.
+class RewardsCounter : public rewards_data::RewardsDataCounter {
```

Comment:
I don't understand this class, this seems like something we should be getting from ledger

Reply:
For ClearBrowsingData, all items have there own class (e.g. `HistoryCounter`, `CacheCounter`, etc) that sub from `BrowsingDataCounter`. I've updated this to also derive from `BrowsingDataCounter` and have removed the `RewardsDataCounter` class.